caluma/tests/__snapshots__/test_schema.ambr

```diff
@@ -153,6 +153,7 @@
   
   type AnalyticsOutput {
     records(before: String = null, after: String = null, first: Int = null, last: Int = null): AnalyticsTableContentConnection
+    summary(before: String = null, after: String = null, first: Int = null, last: Int = null): AnalyticsRowConnection
```

Comment:
More of a general question: We extend the standard connection types in analytics which means we don't have a total count and I'm not sure that the standard implementation of the pagination (especially the `PageInfo`) works properly - is that intended?

Reply:
For now, we don't do pagination or total count output, correct. Will add this later on if needed

Reply:
If we don't do pagination, why are we exposing pagination params? Either implement it or omit it..